ext/standard/dir.stub.php

```diff
@@ -87,7 +87,11 @@
  */
 const SCANDIR_SORT_NONE = UNKNOWN;
 
-class Directory
+/**
+ * @strict-properties
+ * @not-serializable
+ */
+final class Directory
```

Comment:
Because the class is now final, I suppose the return types of the methods no longer need to be tentative?

Reply:
Actually a good point, will fix this.